### PR TITLE
[SRE-160] CDN Redirect - add missing prefix

### PR DESCRIPTION
**Pull request by pwilczynskiclearcode**

### CODE REVIEW

File: handlers/geolocation/geolocation.go [hunk 1]
```diff
@@ -46,7 +46,13 @@ func (c *GeolocationHandlersCollection) RedirectHandler() httprouter.Handle {
 				newURL, _ := url.Parse(r.URL.String())
 				newURL.Scheme = protocol(r)
 				newURL.Host = c.Config.CdnRedirectPrefix.Host
-				newURL.Path, _ = url.JoinPath(c.Config.CdnRedirectPrefix.Path, newURL.Path)
+				fullPlaybackID := playbackID
+				if prefix == "" {
+					fullPlaybackID = fmt.Sprintf("video+%s", fullPlaybackID)
```
Comment (leszko) [inline handlers/geolocation/geolocation.go:51]: What is the case when the `prefix` is an empty string? And why we use `video+` (and not `videorec+`) in that case?

Comment (pwilczynskiclearcode) [reply handlers/geolocation/geolocation.go]: Initially I thought we can avoid `GetBestNode()` node... but querying mist seems to be the only way to extract the correct prefix

**Review by iameli — APPROVED** [review]
Cool!

**Review by leszko — APPROVED** [review]
LGTM 👍

**Pull request merged**